### PR TITLE
Submitting pseudocode

### PR DESCRIPTION
**Pull request by cody-berry**

### CODE REVIEW

File: 2765. Longest Alternating Subarray/solution.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Solution:
```
Comment (cody-berry) [inline 2765. Longest Alternating Subarray/solution.py:1]: Oh no whoops I forgot my function description!

File: 2765. Longest Alternating Subarray/solution.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        
+        # define the previous number starting at the first number
+        
+        # for each number that's not the first...
+        
+            # the subarray is supposed to look like [3,4,3,4,3,...], 
+            # so s[i+1] - s[i] = (-1)^(i+1). 
+            
+                # if it meets the requirements, make the previous
+                # number this number and increment the current
```
Comment (aeruru) [inline 2765. Longest Alternating Subarray/solution.py:15]: what do you mean make the previous number this number? Are you incrementing an index?

**Review by aeruru — APPROVED** [review]

File: 2765. Longest Alternating Subarray/solution.py (hunk 2 above)
Comment (aeruru) [reply 2765. Longest Alternating Subarray/solution.py:15]: Ah, ok I saw your code. Better to save, set the previous number variable to the current number.

Comment (aeruru) [reply 2765. Longest Alternating Subarray/solution.py:15]: Or for the comment when you declare the "previous number", make it clear that that variable is to store the value of the number before the current number so you can diff them. I think index based iteration would have been simpler in this case but this works!

**Pull request merged**